docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
Ok - had to add a SHELL ["powershell", "-command"] line, but otherwise they seem to work.  Testing, then I'll resubmit with the in-lined Linux docker files